src/components/flyout/flyout.spec.tsx

```diff
@@ -95,5 +126,41 @@ describe('EuiFlyout', () => {
           expect(cy.get('[data-test-subj="flyoutSpec"]').should('exist'));
         });
     });
+
+    it('does not interfere when a shard is clicked', () => {
```

Comment:
This is a pretty ambiguous title and ambiguous test. I think I'd prefer it to be more specific, e.g.

```suggestion
    it('does not close the flyout when a shard is clicked', () => {
```

and instead of setting the shard to the flyout toggle and clicking the toggle (which IMO is already tested by EuiCollapsibleNav's specs), just have the shard be some random div or button that does nothing and simply confirm that the flyout did not close because the shard is considered part of the flyout.